common/src/main/java/com/microsoft/identity/common/internal/request/SdkType.java

```diff
@@ -22,10 +22,34 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.request;
 
+import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
+
 /**
  * Enum to indicate if the request came from an app hosting Adal or Msal Sdk
  */
 public enum SdkType {
     ADAL,
-    MSAL
-}
\ No newline at end of file
+    MSAL,
+    MSALCPP;
+
+    /**
+     * Function for mapping the SdkType to appropriate String
```

Comment:
Functions in java are called as `methods`

Reply:
38b8d704207fb078d883d644a73e0835fa8ded39